requirements/default.txt

```diff
@@ -60,9 +64,9 @@ flake8==3.7.9 \
     --hash=sha256:49356e766643ad15072a789a20915d3c91dc89fd313ccd71802303fd67e4deca
 genc==0.2 \
     --hash=sha256:bf568548cd9c30e2a059110eeb33cecc7fb74875cf10e5c897caf6bc762b6221
-geoip2==2.9.0 \
-    --hash=sha256:a37ddac2d200ffb97c736da8b8ba9d5d8dc47da6ec0f162a461b681ecac53a14 \
-    --hash=sha256:f7ffe9d258e71a42cf622ce6350d976de1d0312b9f2fbce3975c7d838b57ecf0
+geoip2==3.0.0 \
```

Comment:
Breaking change in ``geoip2.record.*`` classes, but we just use ``configure_geoip`` ✓